### ISSUE TITLE
Upon Startup, if logs.db is corrupt, regenerate it

### ISSUE THREAD

**Issue by Caleb-C** [Sonarr/Sonarr#1015]
I recently had an issue where Sonarr would not start.  In the log file, I saw this message: "System.Data.SQLite.SQLiteException: database disk image is malformed".  I checked the integrity of the nzbdrone.db and it was fine, but Sonarr still would not start.  I then removed the logs.db and Sonarr started up successfully.

I think that this should not happen for a corrupt logs.db.  If it encounters a problem reading/writing to the logs.db, it should attempt to rename it and generate a new one (while raising the issue in the UI).

**Comment by dannolan** [comment]
I want to add a 👍 to this proposal because I got extremely burned by this earlier on. I also had an EXTREMELY large logs.db (for some reason over 4GB) which was causing an out of memory error, but extremely intermittently. I had to nuke the DB to get Sonarr to work again. I'm happy to do the heavy lifting on this if anyone knows where I'd have to patch it - I haven't dabbled in C# in a while but happy to do some work on it.
